src/plugins/console/public/styles/_app.scss

```diff
@@ -29,11 +29,7 @@
   left: 0;
   bottom: 0;
   right: 0;
-}
-
-.consoleVariablesBottomBar {
```

Comment:
This class is not needed anymore after we added the overflow property to the panel.